lemming.go

```diff
@@ -70,7 +121,26 @@ func New(lis net.Listener, targetName, zapiURL string, opts ...grpc.ServerOption
 		recs = append(recs, dplane)
 	}
 
-	s := grpc.NewServer(opts...)
+	resolvedOpts := resolveOpts(opts)
+
+	jcfg := resolvedOpts.deviceConfigJSON
+	root := &oc.Root{}
+	switch jcfg {
+	case nil:
+		root.GetOrCreateNetworkInstance(fakedevice.DefaultNetworkInstance).Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE
```

Comment:
do we always want to create a default network instance, if even when a config is specified? i'm not sure how the initial is intended to be used

Reply:
The initial config may specify a different-named network instance, which is why this is guarded.

Reply:
There should be one and only one default network instance.